tests/modules/nf-core/bedtools/groupby/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: bedtools groupby test_bedtools_groupby
+  command: nextflow run ./tests/modules/nf-core/bedtools/groupby -entry test_bedtools_groupby -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/bedtools/groupby/nextflow.config
```

Comment:
so which local config do we need to add?
only the `tests/config/nextflow.config`?

Reply:
yes, sorry, was on the dev version of tools (and https://github.com/nf-core/tools/pull/2400 still needs to be merged). will remove the addtional configs again